From: Andreas Stieger <astieger@suse.com>
Upstream: no

Adjust the install paths
Move HTML docs to devel package

Index: editorconfig-core-c-0.12.2/doc/CMakeLists.txt
===================================================================
--- editorconfig-core-c-0.12.2.orig/doc/CMakeLists.txt
+++ editorconfig-core-c-0.12.2/doc/CMakeLists.txt
@@ -125,7 +125,7 @@ if(BUILD_DOCUMENTATION)
 
         if(INSTALL_HTML_DOC)
             install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/html"
-                DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/doc/editorconfig")
+                DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/doc/packages/libeditorconfig-devel")
         endif(INSTALL_HTML_DOC)
 
     else(DOXYGEN_FOUND)
